packages/repository-json-schema/README.md

```diff
@@ -0,0 +1,55 @@
+# @loopback/repository-json-schema
+
+Convert a TypeScript class/model to a JSON Schema for users, leveraging LoopBack4's decorators, metadata, and reflection system.
+
+## Overview
+
+This package provides modules to easily convert LoopBack4 models that have been decorated with `@model` and `@property` to a matching JSON Schema Definition.
+
+## Installation
+
+```shell
+$ npm install --save @loopback/repository-json-schema
+```
+
+## Basic use
+
+```ts
+import {getJsonDef} from '@loopback/repository-json-schema';
+import {model, property} from '@loopback/repository';
+
+@model()
+MyModel {
+  @property() name: string;
+}
+
+const jsonDef = getJsonDef(MyModel);
```

Comment:
Since we are producing JSON Schema, I am proposing to rename `getJsonDef` to `getJsonSchema`, and `jsonDef` to `jsonSchema` or simply `schema`.